packages/web/src/scss/components/SegmentedControl/_theme.scss

```
@@ -0,0 +1,49 @@
+@use '@tokens' as tokens;
+@use '../../settings/transitions' as transitions;
```

Renaming is needless as it doesn't change the name 🙂.

```suggestion
@use '../../settings/transitions';
```